packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -337,10 +331,8 @@ export function publish(options: InitOptions): void {
       }
 
       let callFlowlet: IALFlowlet | undefined;
-      if (shouldPushPopFlowlet(event) && (callFlowlet = activeUIEventFlowlets.get(eventName)) != null) {
+      if (shouldPushPopFlowlet(event)) {
```

Comment:
you can remove this as well. The code in `FlowletWrapper` will take care of push/pop.